osc/build.py

```diff
@@ -675,6 +673,11 @@ def main(apiurl, opts, argv):
         except:
             pass
 
+    if opts.shell:
+        buildargs.append("--shell")
+        if os.path.exists(build_root) and not opts.clean:
+            buildargs.append("--noinit")
```

Comment:
opts.noinit must be set to True (otherwise the buildinfo and the buildconfig will be downloaded from the server again.)
opts.offline is also needed if the build_root exists.

Reply:
done